TYPO3.Media/Classes/TYPO3/Media/Validator/ImageSizeValidator.php

```diff
@@ -20,7 +20,7 @@
  * Validator that checks size (resolution) of a given image
  *
  * Example:
- * [at]Flow\Validate("$image", type="\TYPO3\Media\Validator\ImageSizeValidator", options={ "minimumWidth"=150, "maximumResolution"=60000 })
+ * [at]Flow\Validate("$image", type="\Neos\Media\Validator\ImageSizeValidator", options={ "minimumWidth"=150, "maximumResolution"=60000 })
```

Comment:
Even though this is just a comment, it will lead people to the use of a not (yet) existing class. So, please only merge with the actual vendor name change.